FEATURES.md

```diff
@@ -422,6 +423,24 @@ tasks:
     namespace: carts
 ```
 
+### Job security context
+
+By default the jobs will use the default security context, which was specified at the time of the installation of the 
+job-executor-service. The job configuration provides a way to overwrite this context on a task level:
+
+```yaml
+tasks:
+  - name: "Run as different user"
+    securityContext:
+      runAsUser: 7000
+      runAsGroup: 9000
+    ...
+  - name: "Allow modifications to root FS"
+    securityContext:
+      readOnlyRootFilesystem: false
+    ...
+```
```

Comment:
Please extend both parts with image and command, whereas with the readOnlyRootfilesystem false the command could be an `echo something > test.txt`